packages/stores/types.ts

```diff
@@ -83,8 +83,12 @@ type TNotification =
     | ((excluded_until: number) => TNotificationMessage);
 
 type TClientStore = {
+    fetchResidenceList: () => any;
+    fetchStatesList: () => any;
+    getChangeableFields: () => any;
     accounts: { [k: string]: TAccount };
     active_accounts: TActiveAccount[];
+    account_settings: object;
```

Comment:
`import { GetSettings } from '@deriv/api-types';`

```suggestion
    account_settings: GetSettings;
```